Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -619,9 +621,17 @@ def merge(args:)
         path.atomic_write(formula_ast.process)
 
         unless args.no_commit?
-          Utils::Git.set_name_email!
+          Utils::Git.set_name_email!(committer: args.committer.blank?)
           Utils::Git.setup_gpg!
 
+          if (committer = args.committer)
+            committer = Utils.parse_author(committer)
+            raise "Unable to parse committer name and email." if committer.blank?
```

Comment:
```suggestion
            raise UsageError, "Unable to parse committer name and email." if committer.blank?
```

But given it's repeated in three places: maybe make this `Utils.parse_author!` and make it throw the exception for you?